src/main/java/com/iota/iri/network/pipeline/SolidifyStage.java

```diff
@@ -0,0 +1,63 @@
+package com.iota.iri.network.pipeline;
+
+import com.iota.iri.controllers.TipsViewModel;
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.model.Hash;
+import com.iota.iri.service.validation.TransactionSolidifier;
+import com.iota.iri.service.validation.TransactionValidator;
+import com.iota.iri.storage.Tangle;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.iota.iri.controllers.TransactionViewModel.fromHash;
+
+public class SolidifyStage implements Stage {
+    private static final Logger log = LoggerFactory.getLogger(SolidifyStage.class);
+
+    private TransactionSolidifier txSolidifier;
+    private TransactionValidator txValidator;
+    private TipsViewModel tipsViewModel;
+    private Tangle tangle;
+
+    public SolidifyStage(TransactionSolidifier txSolidifier, TransactionValidator txValidator,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Public method and constructor comments are required](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4339340)